bindata/bootkube/config/bootstrap-config-overrides.yaml

```diff
@@ -49,13 +49,8 @@ apiServerArguments:
     - /etc/kubernetes/secrets/etcd-client.key
   etcd-servers: {{range .EtcdServerURLs}}
     - {{.}}{{end}}
-  feature-gates:
-    - "APIPriorityAndFairness=true"
-    - "RotateKubeletServerCertificate=true"
-    - "DownwardAPIHugePages=true"
-{{- if .ServiceCIDR | len | eq 2}}
-    - "IPv6DualStack=true"
```

Comment:
Won't this feature gate be missing with the new code?

Reply:
IPv6 was forced to always true in 1.24 and removed in 1.25: https://github.com/kubernetes/kubernetes/blob/release-1.24/pkg/features/kube_features.go#L945